src/pudl/metadata/resources/eia930.py

```diff
@@ -192,6 +192,33 @@
         "etl_group": "eia930",
         "create_database_schema": False,
     },
+    "out_eia930__hourly_demand": {
```

Comment:
I think the name of this table should make it clear that this isn't the generic place to go for all demand -- it's just the aggregations. Also, in the baseline data, is there no aggregation of any of the other unimputed variables for use alongside the demand? Is there a good reason not to also provide aggregated (though not yet imputed) net generation and demand forecasts?

```suggestion
    "out_eia930__hourly_aggregated_demand": {
```